README.md

```diff
@@ -43,7 +44,56 @@ This Home Assistant integration tracks your parcels by fetching data from InPost
 3. **Restart Home Assistant**.
 4. Execute steps **6, 7, and 8** from the HACS installation method above.
 
----
+## Usage Examples
```

Comment:
Do you think it would be beneficial to add a banner on the top of README to indicate that `0.3.0` has major changes and the integration re-install might be needed?

Reply:
Added 
<img width="1007" height="371" alt="image" src="https://github.com/user-attachments/assets/215b76ba-3687-4d99-ad9c-fdd5d0ab97d0" />